sdb/command.py

```diff
@@ -113,59 +141,390 @@ def __init_subclass__(cls, **kwargs):
         """
         super().__init_subclass__(**kwargs)
         for name in cls.names:
-            sdb.register_command(name, cls)
+            register_command(name, cls)
 
     def call(self,
              objs: Iterable[drgn.Object]) -> Optional[Iterable[drgn.Object]]:
         # pylint: disable=missing-docstring
         raise NotImplementedError
 
-    def _call_and_yield(self,
-                        objs: Iterable[drgn.Object]) -> Iterable[drgn.Object]:
+    def call_and_yield(self,
```

Comment:
While you're making lots of changes, would it be reasonable to rename this to `call`, and rename the existing `call` to `_call` (or `_call_impl`, or something)?

Matt and I briefly discussed doing something like this.. so external consumers would use `call` and that would always return an iterable.. rather than having two external method, `call` and `call_and_yield`?

Reply:
Makes sense will do that